modules/custom/d_p_banner/templates/paragraph--d-p-banner.html.twig

```diff
@@ -46,6 +46,7 @@
 {% set paragraph_classes = [
   'd-p-banner',
   'background-image',
+  'd-p-banner--demo',
```

Comment:
temporary usage

Reply:
Tak jak wczoraj rozmawialiśmy - zrobienie tego jako modifiera jest ok, nazwijmy go tylko inaczej, niż `demo` - może `half-transparent`?

Kwestia jak aktywować ten modifier jest jeszcze do przegadania. Jeśli nie będzie na razie dodatkowych settingsów w paragrafie to zrobimy jakiś tymczasowy hook w `d_demo` który wstawi tę klasę.